### ISSUE TITLE
Misconfigured default branches for dependency repositories owned by CHIP-SPV

### ISSUE THREAD

**Issue by JayFoxRox** [CHIP-SPV/chipStar#604]
The default branches on many CHIP-SPV repositories are misconfigured:
- When sending a pull-request, GitHub will want to send it to the default branch.
- When cloning a repository, GitHub will offer to only clone the default (to make it easier to diff in clones and to avoid cluttering the workspace).
- When browsing code on GitHub, it will display the default branch.
- When using the search, it will find content in the default branch.

As a result, currently, it's very hard to follow the development or to contribute to these repositories:

- https://github.com/CHIP-SPV/HIP
    - Default branch should be `chipStar`
    - There are 78 branches, most of which are probably not needed / confusingly named
- https://github.com/CHIP-SPV/HIPCC
    - Default branch should be `chip-spv-develop` (rename to `chipStar`?)
    - There are 9 branches, most of which are probably not needed / confusingly named
- https://github.com/CHIP-SPV/llvm-project
    - Default branch should be `chipspv-llvm-16-patches` (rename to `chipStar-llvm-16`?)
    - There are 47 branches, most of which are probably not needed / confusingly named
- https://github.com/CHIP-SPV/SPIRV-LLVM-Translator
    - Default branch should be `chipspv-llvm-16-patches` (rename to `chipStar-llvm-16`?)
    - There are 16 branches, most of which are probably not needed / confusingly named

Please set the default branch (potentially renaming it) and remove the unnecessary branches.

**Comment by pjaaskel** [comment]
Thanks for the feedback. I suppose now it's as good time as any for a early Fall cleanup. I renamed and set the main branches. We'll clean up the obsolete branches soon.

**Comment by pvelesko** [comment]
The default branches for the CHIP-SPV dependency repos have been renamed and set (per maintainer). Closing; any remaining obsolete-branch cleanup is routine housekeeping.
